test cases/common/1 trivial/meson.build

```diff
@@ -19,6 +19,14 @@ if meson.is_cross_build()
   test('native exe in cross build', native_exe)
 endif
 
+echo = find_program('echo')
```

Comment:
Using shell commands in tests is forbidden. This would need to be a e.g. Python script instead.

This test also does not run the test with coverage enabled so it's not really testing this case FWICT.

Finally, since this is getting fairly complex, it should not be in the test case of `1 trivial`, which is all about testing the simplest of cases.